index.html

```diff
@@ -11,6 +11,7 @@
     <link data-trunk rel="copy-dir" href="webfonts/" />
     <link rel="stylesheet" href="/css/bulma.min.css"/>
     <link rel="stylesheet" href="/css/fas.min.css">
+    <link rel="stylesheet" href="/css/pretty-checkbox.min.css"/>
```

Comment:
Do we really need it? As for me Bulma checkbox is ok too: https://bulma.io/documentation/form/checkbox/

Reply:
Idk. It's pretty but 18kb size

Reply:
Maybe there is a way to leave only used styles for my switch checkbox. I'll look into it.

Reply:
No, that's not a problem. My problem was that I looked at https://ss58.org/pr-31/, but the CSS file hasn't been merged yet; therefore, I didn't see the difference. It was my mistake, sorry.